genomicFileHandler/genomic_file_handlers.py

```diff
@@ -187,51 +167,51 @@ def SM(self):
 
 def faiordict2contigorder(file_name, file_format):
     '''Takes either a .fai or .dict file, and return a contig order dictionary, i.e., chrom_seq['chr1'] == 0'''
-    
+
     assert file_format in ('fai', 'dict')
-    
+
     contig_sequence = []
     with open(file_name) as gfile:
         line_i = gfile.readline().rstrip('\n')
-        
+
```

Comment:
Can be simplified as 
`for line in gfile:`
&nbsp;&nbsp;&nbsp;`line_i = line.rstrip("\n")` so no need to set `line_i` at the end of the loop.